resources/common.rb

```diff
@@ -0,0 +1,27 @@
+actions :install
+default_action :install
+
+# Creates the user if they do not already exist
+attribute :create_user, 'kind_of' => [TrueClass, FalseClass], :default => false
+
+# Create the owner, path, or group if they do not exist
+attribute :path, 'kind_of' => String, :default => '/opt/app/'
```

Comment:
But since this is a resource, what happens if you create two resources without changing the default? The better option is to utilize the ``name_attribute`` as a part of the path (i.e. it should become ``/opt/<name>``. Although I'm not sure you do that in the context of an LWRP, but at least you know the default path will always change since we don't have two resources with the same name.